compiler/rustc_codegen_rmc/src/overrides/hooks.rs

```diff
@@ -231,15 +232,11 @@ impl<'tcx> GotocHook<'tcx> for Nevers {
         _target: Option<BasicBlock>,
         span: Option<Span>,
     ) -> Stmt {
-        let loc = tcx.codegen_span_option(span);
-        // _target must be None due to how rust compiler considers it
-        Stmt::assert_false(
-            &format!(
-                "a panicking function {} is invoked",
-                with_no_trimmed_paths(|| tcx.tcx.def_path_str(instance.def_id()))
-            ),
-            loc,
-        )
+        let msg = format!(
+            "a panicking function {} is invoked",
```

Comment:
I wonder if we can give a clearer message here

Reply:
Sure. Can I do it in a follow up PR since this will likely impact a lot of expected tests. I think this PR is already big enough.